ironic.conf

```diff
@@ -5,6 +5,7 @@ default_boot_interface = ipxe
 default_deploy_interface = direct
 default_inspect_interface = inspector
 default_network_interface = noop
+enabled_bios_interfaces = no-bios,redfish
```

Comment:
Maybe enable the rest of them? At least `idrac-redfish` and `irmc` seem worth enabling